### PR TITLE
readonlys missing in service declarations

### PR DESCRIPTION
**Pull request by william-keller**
readonlys missing in service declarations

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet-architecture/eShopOnContainers?pullRequest=1343) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet-architecture/eShopOnContainers?pullRequest=1343) before we can accept your contribution.<br/><br/>:x: william-keller [sign now](https://cla.dotnetfoundation.org/dotnet-architecture/eShopOnContainers?pullRequest=1343)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet-architecture/eShopOnContainers?pullRequest=1343) it.</sub>

File: src/ApiGateways/Mobile.Bff.Shopping/aggregator/Startup.cs [hunk 1]
```diff
@@ -100,6 +100,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerF
 
     public static class ServiceCollectionExtensions
     {
+        [Obsolete]
```
Comment (mvelosop) [inline src/ApiGateways/Mobile.Bff.Shopping/aggregator/Startup.cs:103]: Please remove as per comment in https://github.com/dotnet-architecture/eShopOnContainers/pull/1344

**Review by mvelosop — CHANGES_REQUESTED** [review]
Hi @william-keller, thanks for the PR.

Just one minor comment.

File: src/ApiGateways/Mobile.Bff.Shopping/aggregator/Startup.cs (hunk 1 above)
Comment (william-keller) [reply src/ApiGateways/Mobile.Bff.Shopping/aggregator/Startup.cs:103]: > Please remove as per comment in #1344

ok @mvelosop

**Comment by william-keller** [comment]
> Hi @william-keller, thanks for the PR.
> 
> Just one minor comment.

I need to remove #1343 too?

**Comment by jamesmontemagno** [comment]
The Xamarin stuff has been moved and is now .NET MAUI migrated.

**Pull request closed without merging**